src/utils/domUtils.ts

```diff
@@ -64,30 +100,50 @@ export module Utils {
       return selection;
     }
 
-    export function boxesOverlap(boxA: ClientRect, boxB: ClientRect) {
-      if (boxA.right < boxB.left) { return false; }
-      if (boxA.left > boxB.right) { return false; }
-      if (boxA.bottom < boxB.top) { return false; }
-      if (boxA.top > boxB.bottom) { return false; }
+    /**
+     * Checks if the first ClientRect overlaps the second.
+     * 
+     * @param {ClientRect} clientRectA The first ClientRect
+     * @param {ClientRect} clientRectB The second ClientRect
+     * @returns {boolean} If the ClientRects overlap each other.
+     */
+    export function clientRectsOverlap(clientRectA: ClientRect, clientRectB: ClientRect) {
+      if (clientRectA.right < clientRectB.left) { return false; }
+      if (clientRectA.left > clientRectB.right) { return false; }
+      if (clientRectA.bottom < clientRectB.top) { return false; }
+      if (clientRectA.top > clientRectB.bottom) { return false; }
       return true;
     }
 
-    export function boxIsInside(inner: ClientRect, outer: ClientRect) {
+    /**
+     * Checks if the first ClientRect is inside the second.
+     * 
+     * @param {ClientRect} clientRectA The first ClientRect
+     * @param {ClientRect} clientRectB The second ClientRect
+     * @returns {boolean} If the first ClientRect is inside the second.
+     */
+    export function clientRectInside(clientRectA: ClientRect, clientRectB: ClientRect) {
```

Comment:
No, because people using Typescript should have tooling available to help them out there.

Javascript consumers would probably be looking at the docs or the d.ts file, otherwise the parameter name wouldn't be available to help them anyway.